apps/synapse-portal-framework/src/pages/FileEntityPage/FileEntityPage.tsx

```diff
@@ -1,13 +1,13 @@
 import { useGetPortalComponentSearchParams } from '@/utils/UseGetPortalComponentSearchParams'
-import { Container, Stack } from '@mui/material'
-import { ProvenanceGraph } from 'synapse-react-client'
+import { Box, Container, Stack } from '@mui/material'
+import { IconSvg, ProvenanceGraph } from 'synapse-react-client'
 import MarkdownSynapse from 'synapse-react-client/components/Markdown/MarkdownSynapse'
 import { useGetEntityBundle } from 'synapse-react-client/synapse-queries'
-import CollapsibleSection from '@/components/CollapsibleSection'
 import { DetailsPageSectionLayoutType } from '@/components/DetailsPage/DetailsPageSectionLayout'
 import { SynapseSpinner } from 'synapse-react-client/components/LoadingScreen/LoadingScreen'
-import DetailsPageMenu from '../../components/DetailsPageMenu'
-import DetailsPageLayout from '@/components/DetailsPageLayout'
+import HeaderCard from 'synapse-react-client/components/HeaderCard'
+import CitationPopover from 'synapse-react-client/components/CitationPopover'
+import { DetailsPageContent } from '../../components/DetailsPage/DetailsPageContentLayout'
 import SynapseFileEntityLinkCard from './SynapseFileEntityLinkCard'
 
 function FileEntityPage() {
```

Comment:
If we need to change the styles (but not the layout/DOM) of the header card, e.g. to give the title a different font size, we could apply styles to this component that target those elements.